.circleci/config.yml

```diff
@@ -428,6 +428,26 @@ jobs:
           paths:
             - .tox
 
+  influx:
+    docker:
+      - *test_runner
+      - image: influxdb:1.5-alpine
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+              - tox-cache-influx-{{ checksum "tox.ini" }}
+      - run: TOX_SKIP_DIST=False tox -e '{py27,py34,py35,py36}-influx{50,51,52}' --result-json /tmp/influx.results
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - influx.results
+      - save_cache:
```

Comment:
Same as `restore_cache` above.

Reply:
Changed in b136a73d087d7dcd31a490335d6a3c7d3bfaf6a0